beacon-chain/execution/payload_body_test.go

```diff
@@ -0,0 +1 @@
+package execution
```

Comment:
Please add tests :rofl:

Reply:
It's a todo in the description of the PR :P

Reply:
I added tests :)